### DISCUSSION TITLE
suppress logs from route

### DISCUSSION THREAD

**Discussion by ep4sh** [go-chi/chi#857] [Q&A] [answered]
Dear community, is there any chance to suppress logs from a specific route?
For instance, I have `/healthz` endpoint in Kubernetes and I do want to avoid logs from it, when using logging middleware.

**Comment by VojtechVitek** [comment, marked as answer]
```go
r.Use(middleware.Heartbeat("/healtz")) // no logger here yet
r.Use(middleware.Logger))              // defined after Heartbeat
r.Use(middleware.Recoverer))           // defined after Logger
```

**Reply by ep4sh** [reply]
Thanks a lot, will try this!
